integration/test-integration/src/test/resources/openAPIs/aws_lambda_openAPI.yaml

```diff
@@ -0,0 +1,132 @@
+openapi: 3.0.1
+info:
+  title: AwsLambda
+  version: 1.0.0
+servers:
+  -
+    url: /
+security:
+  -
+    default: []
+x-wso2-disable-security: false
+paths:
+  /*:
+    get:
+      responses:
+        "200":
+          description: OK
+      security:
+        -
+          default: []
+      x-auth-type: Application & Application User
+      x-throttling-tier: Unlimited
+      x-amzn-resource-name: arn:aws:lambda:us-east-1:123456789:function:testFunc
+      x-wso2-application-security:
+        security-types:
+          - oauth2
+        optional: false
+    put:
+      responses:
+        "200":
+          description: OK
+      security:
+        -
+          default: []
+      x-auth-type: Application & Application User
+      x-throttling-tier: Unlimited
+      x-amzn-resource-name: arn:aws:lambda:us-east-1:123456789:function:testFunc
+      x-wso2-application-security:
+        security-types:
+          - oauth2
+        optional: false
+    post:
+      responses:
+        "200":
+          description: OK
+      security:
+        -
+          default: []
+      x-auth-type: Application & Application User
+      x-throttling-tier: Unlimited
+      x-amzn-resource-name: arn:aws:lambda:us-east-1:123456789:function:testFunc
+      x-wso2-application-security:
+        security-types:
+          - oauth2
+        optional: false
+    delete:
+      responses:
+        "200":
+          description: OK
+      security:
+        -
+          default: []
+      x-auth-type: Application & Application User
+      x-throttling-tier: Unlimited
+      x-amzn-resource-name: arn:aws:lambda:us-east-1:123456789:function:testFunc
+      x-wso2-application-security:
+        security-types:
+          - oauth2
+        optional: false
+    patch:
```

Comment:
if possible we can remove the unnecessary operations relevant to the new test cases.

Reply:
Fixed by e0f92a0